project1/styles.css

```
@@ -0,0 +1,66 @@
+body {
+    background-color: lightgrey;
+    margin: 26px auto;
+    max-width: 800px;
+}
+
+/* === title ===*/
+
+h1 {
+    background-color: black;
+    color: white;
+    font-family: 'Merriweather', serif;
+    font-size: 40px;
+    font-style: oblique;
+    font-weight: 700;
+    text-align: center;
+    text-transform: uppercase;
+    letter-spacing: 4px;
+}
+
+/* === date ===*/
+
+h2 {
```

Instead of repeating so much between the `h2` and the `.author` selectors, combine them up here like so:
```suggestion
h2, .author {
```
Then you can simplify the `.author` to be just the `display: block;` rule.